docs/cri/development.md

```diff
@@ -0,0 +1,49 @@
+# Getting Started for Developers
+
+This document provides steps to build `cri` plugin and test it locally.
+
+> NOTE: This is part of the migration from: https://github.com/containerd/cri#getting-started-for-developers
+
+## Install Dependencies
+
+1. Follow this [guide](https://github.com/containerd/containerd/blob/master/BUILDING.md) and install required dependencies.
+2. Install [CNI](https://github.com/containernetworking/cni) by following its homepage.
+
+> NOTE: An alternative of installing dependencies is running `sudo make install-cri-deps`. This is not offcially supported yet.
```

Comment:
I'd put the note at the top of the section.. see edited note below..., I'm not sure what you meant by not officially supported? but it does work:
```
NOTE: An alternative to installing dependencies manually is running `sudo make install-cri-deps`. 
```

Reply:
By saying not officially supported, I mean install-cri-deps doesn't always work for users. For example, the install script assumes the location where you clone the containerd repo. https://github.com/containerd/containerd/blob/master/script/setup/install-cni#L24

Reply:
> By saying not officially supported, I mean install-cri-deps doesn't always work for users. For example, the install script assumes the location where you clone the containerd repo. https://github.com/containerd/containerd/blob/master/script/setup/install-cni#L24

I would suggest cloning GitHub repositories in this pattern.. "$GOPATH"/src/github.com/containerd/containerd"
If you like you could add the warning that "for development the following pattern should be used ..."

Reply:
I missed that Derek was using install-cri-deps for release builds... It won't work right for local host.  
So let's make it `install-deps`  I'll push a couple prs to make the new install-deps work.  Cheers.

Reply:
https://github.com/containerd/containerd/pull/5132
^  first commit

Reply:
Cool, thanks. After your change, users can just do `sudo make install-deps`, right?

Reply:
yes install-deps is fine now